### PR TITLE
Update k8s comparison docs, ecosystem and community

### PR DESCRIPTION
**Pull request by mwickett**
🔍 [Preview - homepage](https://nomad-3f4piephc.vercel.app/)
🔍 [Preview - k8s page](https://nomad-3f4piephc.vercel.app/docs/nomad-vs-kubernetes)

Supporting marketing, makes a variety of changes, including:

* Move the Nomad vs Kubernetes pages to `/docs` and update copy
* Add Nomad vs Kubernetes callout on homepage
* Shift `/resources` to `/community`
* Move "Who uses Nomad" to a dedicated page under `/docs`
* Set appropriate redirects

### CODE REVIEW

File: website/pages/docs/nomad-vs-kubernetes/index.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Nomad has been proven to scale to cluster sizes that exceed 10,000 nodes in real-world production environments. It can be deployed across multiple availability zones, regions, and data centers with a single cluster or multiple clusters. Nomad is designed to natively handle multi-cluster deployments without the overhead of running clusters on clusters. This makes it easier to scale the application deployment across multiple datacenters, regions, and clouds with no additional complexity.
+
+Nomad has performed strenuous benchmark on scalability with [1 million container](https://www.hashicorp.com/c1m) challenge in 2016 and [2 million container challenge](https://www.hashicorp.com/c2m) in 2020. These tests are aimed to validate Nomad's architectural design and ensure that Nomad performs performs under the most extreme for users
```
Comment (changli0617) [inline website/pages/docs/nomad-vs-kubernetes/index.mdx:36]:
```suggestion
Nomad has performed strenuous benchmark on scalability with [1 million container challenge](https://www.hashicorp.com/c1m) in 2016 and [2 million container challenge](https://www.hashicorp.com/c2m) in 2020. These tests are aimed to validate Nomad's architectural design and ensure that Nomad performs under the most extreme requirements.
```

File: website/pages/docs/nomad-vs-kubernetes/alternative.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+page_title: Alternative to Kubernetes
+sidebar_title: Alternative to Kubernetes
+description: Nomad as an alternative to Kubernetes
+---
+
+# An alternative to Kubernetes
+
+While known for its goal of automating and simplifying application deployment, an orchestrator itself can be extremely complex to implement and manage. Kubernetes requires significant time and deep understanding to deploy, operate, and troubleshoot. Teams and organizations choose Nomad as an alternative to Kubernetes for its two core strengths:
+
+- Simplicity in usage and maintainability
```
Comment (changli0617) [inline website/pages/docs/nomad-vs-kubernetes/alternative.mdx:12]:
```suggestion
- **Simplicity in usage and maintainability**
```

File: website/pages/docs/nomad-vs-kubernetes/alternative.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+- Flexibility to deploy and manage containerized and non-containerized applications
```
Comment (changli0617) [inline website/pages/docs/nomad-vs-kubernetes/alternative.mdx:14]:
```suggestion
- **Flexibility to deploy and manage containerized and non-containerized applications**
```

File: website/pages/docs/nomad-vs-kubernetes/alternative.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+> “Nomad is easy to cluster up. We converted our Kubernetes deployment manifest to Nomad job files, then tested it. And since it’s a single binary, it’s simple to configure to our specific needs, which eliminates much of the complexity we faced with Kubernetes. More importantly, Nomad’s agnostic infrastructure resource pool and automated workflows let us deploy and manage our containers and apps across on-prem and any private or public cloud environment, which dramatically expands our datacenter options while still meeting our data residency obligations.”
+
+**AmpleOrganics**, Canada's #1 cannabis software company migrated off of Kubernetes to Nomad
+
+> “I’m a complete beginner when it comes to distributed computing and orchestration. Nomad virtually eliminates barriers to entry for developers who don’t have cloud computing expertise and makes it really easy to connect to the cluster, configure it, and run my jobs while having full visibility into the jobs’ status so I can restart them if need be.”
+
+**Autodesk**, Autodesk Research built a scalable, maintenance-free, and multi-cloud orchestration workflow
```
Comment (changli0617) [inline website/pages/docs/nomad-vs-kubernetes/alternative.mdx:32]:
```suggestion
**Autodesk**, Autodesk Research built a scalable, maintenance-free, and multi-cloud orchestration workflow with Nomad
```

File: website/components/features-list/index.jsx [hunk 5]
```diff
@@ -1,7 +1,11 @@
-export default function FeaturesList({ title, items }) {
+export default function FeaturesList({ title, items, intro }) {
   return (
     <div className="g-features-list g-grid-container">
       <h2 className="g-type-display-2">{title}</h2>
+      <div
+        className="intro-container"
+        dangerouslySetInnerHTML={{ __html: intro }}
+      />
```
Comment (mwickett) [inline website/components/features-list/index.jsx:8]: This is a bit of a shoehorn here to support the k8s callout text under the "Why HashiCorp" heading. 

Screen Shot 2020-12-08 at 11 53 57 AM

File: website/data/docs-navigation.js [hunk 6]
```diff
@@ -29,7 +29,7 @@ export default [
       'windows-service',
     ],
   },
-  {category: 'upgrade', content: ['upgrade-specific']},
+  { category: 'upgrade', content: ['upgrade-specific'] },
```
Comment (mwickett) [inline website/data/docs-navigation.js:32]: Linter noise in this file.

File: website/data/docs-navigation.js [hunk 7]
```diff
@@ -419,19 +427,26 @@ export default [
...
+      },
+    ],
   },
   '------------',
+  {
+    category: 'nomad-vs-kubernetes',
+    content: ['alternative', 'supplement'],
+  },
+  'ecosystem',
+  'who-uses-nomad',
```
Comment (mwickett) [inline website/data/docs-navigation.js:449]: Sidebar additions here.

**Review by schmichael — APPROVED** [review]

File: website/pages/docs/nomad-vs-kubernetes/index.mdx [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+## Simplicity
+
+Kubernetes is designed as a collection of more than a half-dozen interoperating services which together provide the full functionality. Coordination and storage is provided by etcd at the core. The state is wrapped by API controllers which are consumed by other services that provide higher level APIs for features like scheduling. Kubernetes supports running in a highly available configuration but is operationally complex to setup.
+
+Nomad is architecturally much simpler. Nomad is a single binary, both for clients and servers, and requires no external services for coordination or storage. Nomad combines a lightweight resource manager and a sophisticated scheduler into a single system. By default, Nomad is distributed, highly available, and operationally simple.
+
+## Flexible Workload Support
+
+While Kubernetes is specifically focused on Docker, Nomad is more general purpose. Nomad supports virtualized, containerized and standalone applications, including Docker, Java, IIS on Windows, Qemu, etc. Nomad is designed with extensible drivers and support will be extended to all common drivers.
```
Comment (schmichael) [inline website/pages/docs/nomad-vs-kubernetes/index.mdx:22]: Wording here might use a little update given Kubernetes switching off of "Docker proper" to containerd/CRI.

File: website/pages/docs/nomad-vs-kubernetes/index.mdx [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+In contrast to Kuberentes' fragmented distributions, Nomad as a single lightweight binary can be deployed in local dev, production, on-prem, at the edge, and in the cloud in a consistent manner, and provides the same operational ease-of-use across all environments.
+
+## Scalability
+
+[Kubernetes documentation](https://kubernetes.io/docs/setup/best-practices/cluster-large/) states that they support clusters up to 5,000 nodes and 300,000 total containers. As the environment grows, the interoperating components with different constraints compound the operational complexity. [Even operators at Google revealed the significant challenges of managing the system at scale](https://blog.dave.tf/post/new-kubernetes/). The lack of maturity in the Federation project and the additional overhead of managing a centralized management plane also make it a hard experience to deploy a distributed system that spans multiple clusters.
```
Comment (DingoEatingFuzz) [inline website/pages/docs/nomad-vs-kubernetes/index.mdx:32]: For whatever reason, this is on its own line at some widths

Comment (mwickett) [reply website/pages/docs/nomad-vs-kubernetes/index.mdx:32]: Wow, that's funky. I see it on Chrome.

**Pull request merged**